src/compas_fab/robots/robot.py

```diff
@@ -46,11 +46,21 @@ class Robot(object):
     """
 
     def __init__(self, model, artist=None, semantics=None, client=None):
+        self._scale_factor = 1.
         self.model = model
-        self.artist = artist
+        self._artist = None
+        self.artist = artist  # setter and getter (because of scale)
         self.semantics = semantics
         self.client = client  # setter and getter ?
-        self.scale(1.)
+
+    @property
+    def artist(self):
```

Comment:
Although the name is very clear, since this property is important, please add a little docstring to explain it.